src/pages/MakePayment.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <q-page>
+    <q-modal v-model="isOpenedPayment" maximized>
+    <q-btn class="absolute-top-left" color="primary" flat round dense icon="reply" @click="closePayment()"></q-btn>
+    <div>
+        <div class="q-title row justify-center q-mt-sm text-grey">{{getCustomer.name}}</div>
+        <div class="row justify-center q-mt-sm">Give this code to the cashier for payment</div>
+            <div class="q-pa-lg">
+                <et-gen-q-r-code :dark='dark' :qrcode='getQRCodePayment' class="row justify-center"/>
+            </div>
+   <span id="countdown" class="row justify-center"></span>
+   <q-btn :loading="getIsLoading" class="q-mb-xl fixed-center" :disabled="disabled" color="secondary" @click="btnRefresh" label="Get New Code" >
+      <q-spinner-pie slot="loading" size="25px" />
+   </q-btn>
+</div>
+    </q-modal>
+</q-page>
+</template>
+<script>
+import etGenQRCode from '../components/GenQRCode'
+import {mapGetters, mapActions, mapMutations} from 'vuex'
+export default {
+  data() {
+    return {
+      dark: '#00F',
```

Comment:
#00F -> should create a re-usable constant for this value
const VALID_COLOR = '#00F'
const EXPIRED_COLOR = '#F00'